packages/manager/src/features/CloudPulse/Alerts/CreateAlert/GeneralInformation/ResourceMultiSelect.tsx

```diff
@@ -0,0 +1,94 @@
+import { useFormikContext } from 'formik';
+import * as React from 'react';
+
+import { Autocomplete } from 'src/components/Autocomplete/Autocomplete';
+import { useLinodeResourcesQuery } from 'src/queries/cloudpulse/resources';
+interface CloudPulseResourceSelectProps {
+  /**
+   * cluster type selected by the user
+   */
+  cluster: boolean;
+  /**
+   * name used for the component to set formik field
+   */
+  name: string;
+  /**
+   * region selected by the user
+   */
+  region: string | undefined;
+  /**
+   * service type selected by the user
+   */
+  serviceType: string | undefined;
+}
+
+export const CloudPulseMultiResourceSelect = (
+  props: CloudPulseResourceSelectProps
+) => {
+  const resourceOptions: any = {};
+  const { cluster, name, region, serviceType } = { ...props };
+  const formik = useFormikContext();
+  const [selectedResource, setResource] = React.useState<any>([]);
+  const filterResourcesByRegion = (resourcesList: any[]) => {
+    return resourcesList?.filter((resource: any) => {
+      if (region == undefined) {
```

Comment:
strict equality checks

Reply:
Removed the specific function as we were using xFilters to filter the resources and no need for manual filtering